Index: 职场talk/向优秀学习.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## Lab主任 Ken\r\n\r\n### technique\r\n1. always hands on, never give up coding\r\n2. demo first, give people the first good impression\r\n### career staff\r\n1. 跟对领导，及时止损，在错的领导下，干什么都是错，产出难以被有效评估\r\n2. 离职是一种谈判的手段，有负向的影响，但也有积极的一面：争取更多的薪酬，获取更承重的工作\r\n3. 
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/职场talk/向优秀学习.md b/职场talk/向优秀学习.md
--- a/职场talk/向优秀学习.md	(revision 83e2be1df4a9c6ff6f2404a96dc62a29464b7636)
+++ b/职场talk/向优秀学习.md	(date 1706672846965)
@@ -6,4 +6,5 @@
 ### career staff
 1. 跟对领导，及时止损，在错的领导下，干什么都是错，产出难以被有效评估
 2. 离职是一种谈判的手段，有负向的影响，但也有积极的一面：争取更多的薪酬，获取更承重的工作
-3. 
\ No newline at end of file
+3. “欺上瞒下”
+4. 双重标准：宽以待己，严于律人
\ No newline at end of file
